app/components/elements/FontStyles.js

```diff
@@ -205,9 +205,10 @@ export const TopNavLink = Styled(NavLink)`
 
 export const TextLink = Styled(Link)`
   color: ${colors.text.link};
-  text-decoration: none;
-  line-height: ${lineHeights[0]};
+  font-size: ${fontSizes[1]}px;
```

Comment:
Should we specify a font size?  Would it be better to omit it and let the link size be determined by the parent?